frontend/src/scenes/insights/InsightDisplayConfig.tsx

```diff
@@ -120,44 +125,66 @@ export function InsightDisplayConfig({ filters, activeView, disableTable }: Insi
                 !filters.compare &&
                 (!filters.display || filters.display === ChartDisplayType.ActionsLineGraph) &&
                 featureFlags[FEATURE_FLAGS.SMOOTHING_INTERVAL] ? (
-                    <SmoothingFilter />
+                    <span className="space-x-2 flex items-center text-sm">
+                        <SmoothingFilter />
+                    </span>
                 ) : null}
 
                 {activeView === InsightType.RETENTION && (
-                    <>
+                    <span className="space-x-2 flex items-center text-sm">
                         <RetentionDatePicker />
                         <RetentionReferencePicker />
-                    </>
+                    </span>
                 )}
 
                 {showPathOptions && (
-                    <span className="filter">
+                    <span className="space-x-2 flex items-center text-sm">
                         <PathStepPicker />
                     </span>
                 )}
 
                 {showComparePrevious[activeView] && (
-                    <span className="filter">
+                    <span className="space-x-2 flex items-center text-sm">
                         <CompareFilter />
                     </span>
                 )}
             </div>
-            <div className="display-config-inner-row">
+            <div className="flex items-center space-x-2 flex-wrap mx-0 my-2">
                 {showChartFilter(activeView) && (
-                    <span className="filter">
+                    <span className="space-x-2 flex items-center text-sm">
                         <span className="head-title-item">Chart type</span>
                         <ChartFilter filters={filters} disabled={filters.insight === InsightType.LIFECYCLE} />
                     </span>
                 )}
+                {activeView === InsightType.TRENDS && (
+                    <span className="space-x-2 flex items-center text-sm">
+                        <span>y-axis format</span>
+                        <LemonSelect
+                            value={filters.y_axis_format || 'numeric'}
+                            onChange={(value) => {
+                                if (isYAxisFormat(value)) {
```

Comment:
Why do we need this? It seems odd that an invalid format would be selectable

Reply:
It's only to coerce the type from maybe string to literal value